hal/include/HAL/HAL.h

```diff
@@ -68,7 +68,7 @@ HAL_PortHandle HAL_GetPortWithModule(int32_t module, int32_t channel);
 
 uint64_t HAL_GetFPGATime(int32_t* status);
 
-int32_t HAL_Initialize(int32_t mode);
```

Comment:
I don't think so. Its not an often enough used functions, and overloads in the HAL are not easy/very ugly.